line-bot-model/src/main/java/com/linecorp/bot/model/response/Status.java

```diff
@@ -0,0 +1,21 @@
+package com.linecorp.bot.model.response;
+
+/**
+ * A status of current calculation.
+ */
+public enum Status {
```

Comment:
This enum class can be used in common, but each comment of the enum seems a little different among some responses.
So I think there is no problem each response class has an inner `Status` class.